recipes/cmaes/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import (
+    export_conandata_patches,
+    get,
+    rmdir,
+    rm,
+    copy,
+    apply_conandata_patches,
+)
+from conan.tools.build import check_min_cppstd
+import os
+
```

Comment:
```suggestion

required_conan_version = ">=2.0"
```

Make sure your new recipe is Conan 2.x compatible only. Conan Center Index no longer validate Conan 1.x

Reply:
The CI can build it with conan 2. Is this enough or are there some additional checks I can run?

Reply:
Sorry, I meant this attribute makes sure this recipe requires Conan 2.x at least: https://docs.conan.io/2/reference/conanfile/attributes.html#required-conan-version